### PR TITLE
Add View Command

### PR DESCRIPTION
**Pull request by meerkatboy**
Add View Command for users to view application details of a company specified by its index

### CODE REVIEW

File: src/test/java/seedu/internbuddy/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -57,7 +57,6 @@ public class CommandTestUtil {
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "invalid!email"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
-    public static final String INVALID_STATUS_DESC = " " + PREFIX_STATUS;
```
Comment (realqijun) [inline src/test/java/seedu/internbuddy/logic/commands/CommandTestUtil.java:60]: What does this do

File: src/main/java/seedu/internbuddy/logic/parser/FindCommandParser.java [hunk 2]
```diff
@@ -15,7 +16,7 @@
  */
 public class FindCommandParser implements Parser<FindCommand> {
 
-    private static Logger logger = Logger.getLogger("Foo");
+    private static Logger logger = LogsCenter.getLogger(FindCommand.class);
```
Comment (realqijun) [inline src/main/java/seedu/internbuddy/logic/parser/FindCommandParser.java:19]: Good catch!

**Review by realqijun — APPROVED** [review]
Approved! 🚀 This code is so smooth, it might just be the next TikTok dance trend 💃🕺
Great job implementing the ViewCommand and creating tests for most of the stuff! Let's get this code up and running!

File: src/test/java/seedu/internbuddy/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (meerkatboy) [reply src/test/java/seedu/internbuddy/logic/commands/CommandTestUtil.java:60]: It wasn't used, so I removed it from the code!

File: src/main/java/seedu/internbuddy/model/company/Company.java [hunk 3]
```diff
@@ -45,6 +46,7 @@ public Company(Name name, Phone phone, Email email, Address address, Set<Tag> ta
         this.status = status;
         this.applications = applications;
         this.isFavourite = isFavourite;
+        this.isLong = isLong;
```
Comment (uniqly) [inline src/main/java/seedu/internbuddy/model/company/Company.java:49]: I think `isLong` is not a very good name for this variable, why not call it `isShowingDetails` for example?

File: src/main/java/seedu/internbuddy/logic/commands/ViewCommand.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            throw new CommandException(Messages.MESSAGE_INVALID_COMPANY_DISPLAYED_INDEX);
+        }
+
+        Company companyToView = lastShownList.get(index.getZeroBased());
+        CompanyToViewPredicate predicate = new CompanyToViewPredicate(companyToView);
+
+        Company editedCompany = new Company(companyToView.getName(), companyToView.getPhone(), companyToView.getEmail(),
+                companyToView.getAddress(), companyToView.getTags(), companyToView.getStatus(),
+                companyToView.getApplications(), companyToView.getIsFavourite(), true);
+        model.setCompany(companyToView, editedCompany);
```
Comment (uniqly) [inline src/main/java/seedu/internbuddy/logic/commands/ViewCommand.java:54]: Perhaps this can be abstracted out?

**Review by uniqly — CHANGES_REQUESTED** [review]
I don't see any way for the detailed view to be reverted after running other commands. I suggest that you add a `Model#hideAllDetails()` and `Model#showDetails(Company)` methods instead and within `LogicManager#execute()` execute `setAllDetailsLow()` at the start of the method.

**Pull request merged**